app/commands/scrape_command.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class ScrapeCommand
```

Comment:
While there is nothing wrong with command objects, it is best to restrict yourself to the default objects listed in the project structure: https://nimblehq.co/compass/development/code-conventions/ruby/ruby-on-rails/#project-structure. The team's convention is to have only service objects, so if a service can call other objects, e.g., `ScrapService` can call the `Search` and `Parse` services. 

If the command pattern is something you want to introduce to the team, you can do it. You must follow [the change management guidelines](https://nimblehq.co/compass/intro/change-management/). It will likely more time though, due to the team sizes to get an agreement on this before we can merge this PR. So it is better to stick to the current conventions.

Reply:
Fixed in https://github.com/Goose97/google-scraper-ruby/pull/20/commits/9c496ceb5b69eeff260558d97ff0aa5179300d82